src/main/java/hudson/plugins/performance/PerformancePublisher.java

```diff
@@ -1221,10 +1215,6 @@ public boolean isPRT() {
         return localReports;
     }
 
-    public static String getOptionType() {
```

Comment:
So far as I can tell, it literally had no purpose other than a really, really broken attempt to make sure a selection persisted in the UI. Except it didn't work, since it was getting changed by any instantiation of `PerformancePublisher` with a different value for `configType`. It was...weird.